packages/react/src/feature-link-list/FeatureLinkListItem.tsx

```diff
@@ -0,0 +1,70 @@
+import { ReactNode } from 'react';
+import { LinkProps, packs } from '../core';
+import { TextLink, TextLinkExternal } from '../text-link';
+import { Box } from '../box';
+import { Flex } from '../flex';
+import { Stack } from '../stack';
+import { Text } from '../text';
+import { ArrowRightIcon } from '../icon';
+import { FeatureLinkListBackground, hoverColorMap } from './utils';
+
+export type FeatureLinkListItemProps = LinkProps & {
+	label: ReactNode;
+	secondaryLabel: ReactNode;
+	background?: FeatureLinkListBackground;
+};
+
+export const FeatureLinkListItem = ({
+	label,
+	secondaryLabel,
```

Comment:
I think this should be `description` which would be consistent with lots of other secondary content in components